### PR TITLE
Feature/reviews lists

### PR DESCRIPTION
**Pull request by dispatchrabbi**

### CODE REVIEW

**Comment by dispatchrabbi** [comment]
This branch was off of the `feature/trello-integration` branch, so the diff for this will get much nicer once that's been merged.

**Comment by dispatchrabbi** [comment]
And on to this one. `npm install` possibly required; note the changes in the README.

File: helpers/format-website.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function formatWebsite(website) {
+  return website.startsWith('http') ? website : 'http://' + website;
```
Comment (kristinstaton) [inline helpers/format-website.js:2]: So, the only website link that's working as intended is Pleasant House Pub. The others aren't working for me. 

- Restaurants on the homepage are linking out to 127.0.0.1:8888/website.com
- Restaurants on the hit list page are linking out to 127.0.0.1:8888/hit-list/website.com
- In both places, Pleasant House Pub is correctly linking out to www.pleasanthousepub.com

Maybe this is the culprit? It seems like you're only appending `http://` to websites if they're coming out of  Trello with the http already in there. 

So, if that's the case, the error makes sense. I entered Pleasant House's website in Trello as `http://www.pleasanthousepub.com`, but in looking at the other restaurant cards, they're all entered in Trello as `www.website.com` without the http. (Follow up: I updated Longman & Eagle's website to include `http://`, and now it works as intended.)

Comment (dispatchrabbi) [reply helpers/format-website.js:2]: This ought to append `http://` to the front of a website if it doesn't already start with it (or `https`). I'll take a look and see what's happening.

Comment (dispatchrabbi) [reply helpers/format-website.js:2]: I missed calling the `formatWebsite` helper on the websites in the restaurant list, which is used on the homepage and on the hit list page. Nice catch, and nice job tracking it down.

**Review by kristinstaton — APPROVED** [review]
The only weird thing I found was the website functionality. J'approve.

**Pull request merged**